src/js/edu-benefits/utils/options-for-select.js

```diff
@@ -0,0 +1,1594 @@
+const countries = [
```

Comment:
If it looks like these can be shared across apps perhaps we can move it out of the edu-benefits directory and somewhere all apps can easily access? Doesn't have to be done in this first pass though, let's just keep in mind once the HCA is merged in and we can move things around more easily.